packages/ui/src/utils/Zorb.ts

```diff
@@ -0,0 +1,209 @@
+/* eslint-disable @typescript-eslint/restrict-template-expressions */
```

Comment:
Really don't like how much bloat this adds to modal for what it acheives. Let's drop Zorb and all of it's dependencies and instead simplify this in the following way:

There are plenty of ideas of how to generate random color from a given string outlined here:
https://stackoverflow.com/questions/3426404/create-a-hexadecimal-colour-based-on-a-string-with-javascript

Let's grab user's address and split it in 2 parts. Use those parts to create 2 random colors and just style div with them so it looks similar to a zorb. i.e. highlight gradient, body gradient and one of the colors used as border.